polyfill/lib/ecmascript.mjs

```diff
@@ -1644,6 +1644,50 @@ export const ES = ObjectAssign({}, ES2019, {
     }
     return round * increment;
   },
+  RoundTime: (hour, minute, second, millisecond, microsecond, nanosecond, increment, unit, roundingMode) => {
+    let quantity = 0;
+    switch (unit) {
+      case 'day':
+        quantity =
+          (((second + millisecond * 1e-3 + microsecond * 1e-6 + nanosecond * 1e-9) / 60 + minute) / 60 + hour) / 24;
```

Comment:
Should we be concerned about IEEE float rounding issues?

Reply:
I think we should, but I'd suggest that's more of a concern for a more production-ready polyfill such as we might make after Stage 3.